### ISSUE TITLE
Crash with OpenTerrainGenerator

### ISSUE THREAD

**Issue by Khaeops** [YaibaToKen/Armory-Expansion#11]
Upon loading chunks in the world I encounter a crash. FoamFix has intercepted the crash and reckons that OpenTerrainGenerator caused the issue, but I have been playing my modded game fine until I added Armory Expansion. Removing Armory Expansion resolves the problem, but it would be nice if the issue could be fixed.

Crashlog:
[crash-2018-06-28_09.03.29-server.txt](https://github.com/YaibaToKen/Armory-Expansion/files/2144856/crash-2018-06-28_09.03.29-server.txt)

**Comment by YaibaToKen** [comment]
I honestly have no clue why this would happen, Armory Expansion adds nothing that could cause this :(

**Comment by Khaeops** [comment]
I really wanted to play with this mod alongside my other mods, so I went ahead and did the old binary search method, enabling half the mods and seeing if it crashed.

I have actually found the real cause of the crash - PlusTiC. I have no idea why.
https://minecraft.curseforge.com/projects/plustic

The crashlog remains the same, but omitting PlusTiC makes the system happy again.

I haven't updated in a long time, so I might go ahead and update these two mods and try it again...
